src/mqt/core/_core/__init__.pyi

```diff
@@ -1 +1,733 @@
-def add(a: int, b: int) -> int: ...
+from typing import ClassVar, Iterator, overload
+
+class Operation:
+    def __init__(self, arg0: int, arg1: list[Operation], /) -> None: ...
+    def acts_on(self, arg: int, /) -> bool: ...
+    def add_depth_contribution(self, arg: list[int], /) -> None: ...
+    def clone(self) -> Operation: ...
+    @property
+    def controls(self) -> set[Control]: ...
+    @controls.setter
+    def controls(self, arg: set[Control]) -> None: ...
```

Comment:
This seems kind of odd. There should be a better way to declare readwrite properties.

Reply:
One could always leave out the setter, but then the information is missing from the stub. This seems to be the way to do it.